imagebitmap-renderingcontext/context-creation-with-alpha.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Canvas's ImageBitmapRenderingContext test</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/scripting.html#the-imagebitmap-rendering-context">
+<script>
+var width = 10;
+var height = 10;
+
+function testCanvas(ctx, r, g, b, a)
+{
+    var color = ctx.getImageData(5, 5, 1, 1).data;
+    assert_array_equals(color, [r, g, b, a]);
+}
+
+promise_test(function() {
+    function consumeImageBitmapAlphaDefault(image) {
+        var dstCanvas = document.createElement('canvas');
+        dstCanvas.width = width;
+        dstCanvas.height = height;
+        var dstCtx = dstCanvas.getContext('bitmaprenderer');
+        dstCtx.transferFromImageBitmap(image);
+
+        var myCanvas = document.createElement('canvas');
+        myCanvas.width = width;
+        myCanvas.height = height;
+        var myCtx = myCanvas.getContext('2d');
+        myCtx.drawImage(dstCanvas, 0, 0);
+        testCanvas(myCtx, 0, 255, 0, 127);
+    }
+
+    var srcCanvas = document.createElement('canvas');
+    srcCanvas.width = width;
+    srcCanvas.height = height;
+    var ctx = srcCanvas.getContext('2d');
+    ctx.fillStyle = 'rgba(0, 255, 0, 0.5)';
+    ctx.fillRect(0, 0, width, height);
+    createImageBitmap(srcCanvas).then(consumeImageBitmapAlphaDefault);
+}, "Test that canvas.getContext('bitmaprenderer') preserves alpha");
```

Comment:
I think the test should go last in this file, and it should entitled "Test that the 'alpha' context creation attribute is true by default."